docs/elements/hx-menu/index.html

```diff
@@ -0,0 +1,25 @@
+---
+title: <hx-menu>
+---
+{% extends 'element.njk' %}
+{% block content %}
+
+<section>
+  <h2 class="hxSectionTitle" id="attributes">Attributes</h2>
+  <dl>
+    <dt>open</dt>
+    <dd>Opens the menu</dd>
+
+    <dt>position</dt>
+    <dd>Positions the menu</dd>
+
+    <dt>relative-to <small>(optional)</small></dt>
+    <dd>
+      ID of an element to relatively position the menu. If not
```

Comment:
s/`element to relatively position the menu`/`element that the menu is positioned relative to`